### PR TITLE
Add tech.hyperjump.monika

### PR DESCRIPTION
**Pull request by dennypradipta**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/hyperjumptech/monika
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by dennypradipta** [comment]
This PR will be drafted as installed Monika couldn't do PING and URL request

image

File: tech.hyperjump.monika.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    build-options:
+      env:
+        npm_config_offline: 'true'
+    build-commands:
+      - unzip monika*.zip
+      - install -D -t /app/bin monika
+      - install -D -t /app/bin node_sqlite3.node
+    sources:
+      - type: file
+        url: https://github.com/hyperjumptech/monika/releases/download/v1.6.13/monika-v1.6.13-linux-x64.zip
```
Comment (hfiguiere) [inline tech.hyperjump.monika.yml:26]: so you need to tell flathub to not build `aarch64`, or build from source so that this is available on `aarch64`

**Comment by nedrichards** [comment]
bot, build tech.hyperjump.monika

**Comment by hfiguiere** [comment]
appstream file is missing

**Comment by dennypradipta** [comment]
Sorry if it's not the right place to ask questions, but I am stuck with ping.

I've added the network permissions, but it still won't run the ping command. Any ideas on how to do it?

**Comment by hfiguiere** [comment]
there is no ping command in the freedesktop-sdk.

You'd have to build it.

**Pull request closed without merging**